app/org/maproulette/models/dal/TaskReviewDAL.scala

```diff
@@ -550,7 +549,8 @@ class TaskReviewDAL @Inject() (
   )(implicit c: Connection = null): (Int, List[Task]) = {
     var orderByClause = ""
     val whereClause = new StringBuilder(
-      "(tasks.bundle_id is NULL OR tasks.is_bundle_primary = true) AND "
+      "(tasks.bundle_id is NULL OR tasks.is_bundle_primary = true) AND " +
```

Comment:
So one thing I want to start doing is if we modify DAL's and it makes sense we should move the one function that we modify into the new framework, and then add integration tests and postman tests along with it as we move it. This way we can slowly move it piecemeal. Some functions would not really be able to do that as they might be to integrated into other pieces of the code, but it would seem like something like this might be a candidate to do that, although looking at it it seems like quite a long function, so up to you.